src/platform/platform_posix.c

```diff
@@ -98,6 +101,17 @@ CxPlatSystemLoad(
     void
     )
 {
+    //
+    // There are a few cases where we might call CxPlatSystemLoad more than
+    // once. Let's guard against that. The caller is responsible for ensuring
+    // it's not called concurrently.
+    //
+    if (CxPlatSystemLoaded) {
```

Comment:
Should we ref count instead?

Reply:
It's difficult to wedge in CxPlatRefInitialize to initialize the ref count that's not supposed to be visible to the caller.

Reply:
Hmm, we actually have to use a ref count. Otherwise, the resources get freed on the first call to unload/uninit while the app might still access the resources after that.

Reply:
used raw ref count